### PR TITLE
get book's cover image

### PR DESCRIPTION
**Pull request by abarkansalim**
I add a function that get the book's cover image link

### CODE REVIEW

**Comment by maria-antoniak** [comment]
Looks great, thank you! A couple thoughts:

1. I don't think we should include this by default (since not everyone will want these images taking up space). Maybe we can add an optional argument to include this.
2. The Goodreads website has changed a lot recently; does this still work?

File: get_books.py [hunk 1]
```diff
@@ -158,6 +166,7 @@ def scrape_book(book_id):
 
     return {'book_id_title':        book_id,
             'book_id':              get_id(book_id),
+            'cover_image':          get_cover_image(soup),
```
Comment (clach04) [inline get_books.py]: Recommend this be renamed, append `_uri` for consistency (see line 172) and clarity.

**Review by clach04 — CHANGES_REQUESTED** [review]

**Review by maria-antoniak — APPROVED** [review]
Looks good, thank you!

**Review by clach04 — APPROVED** [review]
👍 I've not tested/ran this but lgtm :-)

**Pull request merged**